spec/features/favorites/index_spec.rb

```diff
@@ -37,6 +37,10 @@
       click_button "Add to Favorites"
 
       visit "/favorites"
+      
+      expect(page).to have_no_content(roomba.name)
```

Comment:
I like this test -- it's a good catch in case things are being added that shouldn't be.